pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
@yegor256 Does the FindBugs Qulice Maven plugin provide a way to configure exclusions?